### DISCUSSION TITLE
Add support for the SAM jetway connection API

### DISCUSSION THREAD

**Discussion by aeroniemi** [magknight/787aviatorsTracker#802] [Ideas]
see https://stairportscenerieshelp.freshdesk.com/support/solutions/articles/77000217961--aircraft-plugin-developers-how-to-interact-with-sam

would be quite cool to have jetways that connect using our UI rather than SAM

**Comment by Simleybob** [comment]
what about x plane 12 default jetways?

**Reply by aeroniemi** [reply]
they don't currently have an connection api

**Reply by Simleybob** [reply]
Oh right
